Source/CareerLog/CareerLog.cs

```diff
@@ -474,7 +481,9 @@ private CareerLogDto CreateLogDto(LogPeriod logPeriod)
                 constructionFees = logPeriod.OtherFees,
                 otherFees = logPeriod.OtherFees - constructionFees,
                 fundsGainMult = logPeriod.FundsGainMult,
-                numNautsKilled = logPeriod.NumNautsKilled
+                numNautsKilled = logPeriod.NumNautsKilled,
+                kspReputation = logPeriod.KSPReputation,
```

Comment:
Since you named the Headlines field to `headlinesHype` then it's better to use just `reputation` here

Reply:
ksp isnot necessary. Renamed in both CareerLog and LogPeriod